autoPyTorch/utils/common.py

```diff
@@ -156,3 +186,67 @@ def subsampler(data: Union[np.ndarray, pd.DataFrame, scipy.sparse.csr_matrix],
                x: Union[np.ndarray, List[int]]
                ) -> Union[np.ndarray, pd.DataFrame, scipy.sparse.csr_matrix]:
     return data[x] if isinstance(data, (np.ndarray, scipy.sparse.csr_matrix)) else data.iloc[x]
+
+
+def get_hyperparameter(hyperparameter: HyperparameterSearchSpace,
```

Comment:
Can we add a very simple unit testing that has a configuration space like this:
```
X = Integer, from -5, 5
Y = Float from 0 to 1
X = categoritcal ['a', 'b', 1] 
```

And make sure that this work?